src/MLJGLMInterface.jl

```diff
@@ -327,4 +327,78 @@ metadata_model(
     path = "$PKG.LinearCountRegressor"
 )
 
+const DOC_LINEAR_REGRESSION = "TODO"
+"""
+$(MMI.doc_header(LinearRegressor))
+`LinearRegressor` implements the $DOC_LINEAR_REGRESSION.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: any table of input features (eg, a `DataFrame`) whose columns
+  each have one of the following element scitypes: `Continuous`,
+  `Count`, or `<:OrderedFactor`; check column scitypes with `schema(X)`
+
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `Continuous`; check the scitype with `scitype(y)`
+
+# Hyper-parameters
+
+- `fit_intercept=true`: Whether to calculate the intercept for this model.
+   If set to false, no intercept will be calculated (e.g. the data is expected
+   to be centered)
+- `allowrankdeficient=false`: Whether to allow rank deficient matrices (e.g.
+   more columns than rows)
+- `offsetcol=nothing`: Name of the column to be used as an offset, if any.
+   An offset is a variable which is known to have a coefficient of 1.
+# Operations
+
+- `predict(mach, Xnew)`: return predictions of the target given new
+   features `Xnew` having the same Scitype as `X` above. Predictions are
+   probabilistic but uncalibrated.
```

Comment:
I think we should remove "but uncalibrated" here. Unlike DecisionTreeClassifier, the returned distributions really do have a probabilistic interpretation. (For future reference, the usual case for `Probabilistic` models, DecisionTreeClassifier` is an exception.)

Reply:
Good catch!  Will fix?